src/renderer/services/db/api/database.ts

```diff
@@ -2,16 +2,19 @@ import {DatabaseService, IDatabaseApiOptions} from "./abstract";
 import {AccountRepository} from "./repositories/account";
 import {IpcDatabaseController} from "../controller/ipc";
 import {BeaconNodeRepository} from "./repositories/beaconNode";
+import {AttestationsRepository} from "./repositories/attestations";
 
 export class CGDatabase extends DatabaseService {
 
     public account: AccountRepository;
     public beaconNodes: BeaconNodeRepository;
+    public attestations: AttestationsRepository;
```

Comment:
It's probably better to have validatorDB instance and inside of that validator specific repos